File: parent-337/pinyougou-web-shop/src/main/webapp/js/service/goodsPutawayService.js
app.service('goodsPutawayService', function ($http) {
    this.search = function (pageNum,pageSize,searchEntity) {
        return $http.post('../goodsPutaway/search.do?pageNum=' + pageNum + '&pageSize='+ pageSize,searchEntity);
    };

    this.putaway=function (selectIds) {
        return $http.post('../goodsPutaway/putaway.do?selectIds='+selectIds);
    }
    this.sold_out=function (selectIds,isMarketable) {
        return $http.post('../goodsPutaway/sold_out.do?selectIds='+selectIds);
    }

}
);

File: parent-337/pinyougou-interface/src/main/java/cn/ithcast/core/service/GoodsPutawayService.java
package cn.ithcast.core.service;

import cn.ithcast.core.pojo.good.Goods;
import entity.PageResult;
import entity.Result;

public interface GoodsPutawayService {
    PageResult search(Integer pageNum, Integer pageSize, Goods goods,String name);

    void putaway(Long[] selectIds);

    Result sole_out(Long[] selectIds);
}
